### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by amyblais**
For https://community.mattermost.com/core/pl/5acg3tepgbb5uk1as41rwzs8ze

Feel free to edit and add suggestions as much as you want.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -34,3 +34,7 @@ Please include relevant error messages and/or screenshots.
 <!--
 If you can, link to the line of code that might be responsible for the problem.
 -->
+
+---
+
+JIRA tickets are created for all valid issues and then Help Wanted GitHub issues are opened up for those JIRA tickets that our developer teams are not immediately working on. The internal goal is to bring the desktop repo into our day-day process with the hope that this process change will mean more frequent desktop releases with faster feature development and turnaround time on bug fixes.
```
Comment (esethna) [inline ISSUE_TEMPLATE.md]: Might be useful to provide links on the following words:

"JIRA tickets" > link to the JIRA query for desktop issues
"Help Wanted GitHub issues" > link to the process for contributing

Comment (esethna) [inline ISSUE_TEMPLATE.md]: Propose:

```
[JIRA tickets]() are created for all valid bug reports filed here. Bugs and features that the Mattermost core team are not immediately working on are opened as [Help Wanted GitHub issues]() and welcome to contributions. 

Our goal is to bring the Mattermost desktop repository into our day-to-day internal process with the hope of having more frequent desktop releases with faster feature development and turnaround time on bug fixes.

Comment (esethna) [reply ISSUE_TEMPLATE.md]: @wget does this help? Anything you'd like to see here to help with the discussion we've been having in Desktop channel?

**Comment by hanzei** [comment]
Hey @wget,

Just want to check out what do you think about this change? Does it help with the GH vs. JIRA issue?

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (esethna) [reply ISSUE_TEMPLATE.md]: @amyblais we can update to the above text and then I think we're good to merge

Comment (esethna) [reply ISSUE_TEMPLATE.md]: NVM, already updated, thanks!

**Comment by esethna** [comment]
Adding @deanwhillier to review and merge

**Pull request merged**